Don't sys.exit(1) because then we cannot fallback to lsb

Debian Bug #742707
Index: apport/apport/packaging.py
===================================================================
--- apport.orig/apport/packaging.py	2014-03-26 20:08:21.882978340 +0530
+++ apport/apport/packaging.py	2014-03-26 20:09:57.284133160 +0530
@@ -269,7 +269,7 @@
                 return self._os_version
             else:
                 sys.stderr.write('invalid /etc/os-release: Does not contain NAMe and VERSION_ID\n')
-                sys.exit(1)
+                sys.stderr.write('fall back to lsb_release\n')
 
         # fall back to lsb_release
         p = subprocess.Popen(['lsb_release', '-sir'], stdout=subprocess.PIPE,
